src/bot-handlers/tests/liquidator-test.js

```diff
@@ -0,0 +1,497 @@
+/* eslint-disable no-loop-func, max-len */
```

Comment:
Would it be helpful to include a comment at the top of this file that indicates most of this code is taken from UMA source code file xxx?